src/arrays.js

```diff
@@ -8,34 +8,62 @@ const each = (elements, cb) => {
   // Iterates over a list of elements, yielding each in turn to the `cb` function.
   // This only needs to work with arrays.
   // based off http://underscorejs.org/#each
+  for (let i = 0; i < elements.length; i++) { cb(elements[i], i); }
```

Comment:
For readability's sake, I would definitely move `cb(elements[i], i);` to its own line.